src/hydrate.js

```diff
@@ -2,7 +2,7 @@ import { render, hydrate as hydrativeRender } from 'preact';
 
 let initialized;
 
-/** @type {typeof render} */
+/** @type {typeof hydrativeRender} */
```

Comment:
This has always been flat-out wrong; TS will (correctly) report that the function declaration does not match the signature.

`render`, due to `replaceNode`, takes 2-3 parameters, while our little wrapper here only takes 2, ergo, doesn't match the type.